### DISCUSSION TITLE
Spacy dependency issue that started today (via typer and click)

### DISCUSSION THREAD

**Discussion by ggprod** [explosion/spaCy#10566] [Help: Coding & Implementations] [answered]
A service build that depends on spacy recently started failing (earlier today) with an error like the one raised in this issue against typer (which is a dependency of spacy): https://github.com/tiangolo/typer/issues/377 

This is related to an update today of click (8.1.0) that typer depends on.  Once typer is fixed will spacy need any changes to address that?

**Comment by adrianeboyd** [comment, marked as answer]
Please keep an eye on #10564 for updates. Nothing directly in spacy needs to be fixed, the issues are only in typer.
